src/v2i-hub/RSUHealthMonitorPlugin/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "RSUHealthMonitor",
+    "description": "Monitor RSU health status",
+    "version": "@PROJECT_VERSION@",
+    "exeLocation": "/bin/RSUHealthMonitorPlugin",
+    "coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+    "messageTypes": [],
```

Comment:
You should probably add the RSU Health message to message types here.

Reply:
What is the purpose of adding this message types here?

Reply:
I guess in this case it won't work but for plugins that receive a certain message, it adds a status item that includes a count for the number of that type of message received by the plugin.